registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -385,20 +388,20 @@ NormalDistributionsTransform<PointSource, PointTarget>::updateDerivatives (Eigen
       for (Eigen::Index j = 0; j < hessian.cols (); j++)
       {
         // Update hessian, Equation 6.13 [Magnusson 2009]
-        hessian (i, j) += e_x_cov_x * (-gauss_d2_ * x_trans.dot (cov_dxd_pi) * x_trans.dot (c_inv * point_gradient_.col (j)) +
+        hessian (i, j) += e_x_cov_x * (-gauss_d2_ * x_trans.dot (cov_dxd_pi) * x_trans.dot (c_inv * point_jacobian_.col (j)) +
                                     x_trans.dot (c_inv * point_hessian_.block<3, 1>(3 * i, j)) +
-                                    point_gradient_.col (j).dot (cov_dxd_pi) );
+                                    point_jacobian_.col (j).dot (cov_dxd_pi) );
       }
     }
   }
 
-  return (score_inc);
+  return score_inc;
 }
 
 
 template<typename PointSource, typename PointTarget> void
 NormalDistributionsTransform<PointSource, PointTarget>::computeHessian (Eigen::Matrix<double, 6, 6> &hessian,
-                                                                        PointCloudSource &trans_cloud, Eigen::Matrix<double, 6, 1> &)
+                                                                        PointCloudSource &trans_cloud)
```

Comment:
Note: Function signature changed

Reply:
Instead of a hard break, in the header, please deprecate the original function and redirect it to the new one without the unused parameter

Reply:
Done